spec/features/multiwiki_assignment_spec.rb

```diff
@@ -45,38 +45,6 @@
     end
   end
 
-  it 'creates a valid assignment from an article and an alternative project and language' do
```

Comment:
Rather than removing this test, I'd prefer that it be updated to work with the new behavior by adding es.wikisource as a tracked wiki (but not the home wiki) for the test `course`.

Reply:
got it!